packages/snjs/lib/services/component_manager.ts

```diff
@@ -860,30 +890,37 @@ export class SNComponentManager extends PureService {
     });
   }
 
-  handleStreamItemsMessage(component: SNComponent, message: ComponentMessage) {
+  handleStreamItemsMessage(
+    component: SNComponent,
+    message: ComponentMessage
+  ): void {
+    const data = message.data as StreamItemsMessageData;
+    const types = data.content_types
+      .filter((type) => AllowedBatchPermissions.includes(type))
```

Comment:
Main change 1

Reply:
`types` is sorted twice:

- on line `900` and line `904`

Reply:
Good catch 👍